docs/data/scheduler/drag-interactions/DragAndDropResizeStart.tsx

```diff
@@ -0,0 +1,26 @@
+import * as React from 'react';
+import { SchedulerEvent } from '@mui/x-scheduler/models';
+import { EventCalendar } from '@mui/x-scheduler/event-calendar';
+import {
+  initialEvents,
+  defaultVisibleDate,
+  resources,
+} from '../datasets/personal-agenda';
+
+export default function DragAndDropResizeStart() {
+  const [events, setEvents] = React.useState<SchedulerEvent[]>(initialEvents);
+
+  return (
+    <div style={{ height: '600px', width: '100%' }}>
+      <EventCalendar
+        events={events}
+        resources={resources}
+        defaultVisibleDate={defaultVisibleDate}
+        onEventsChange={setEvents}
+        areEventsDraggable
+        areEventsResizable="end"
```

Comment:
```suggestion
        areEventsResizable="start"
```
if we follow the file name